package.json

```diff
@@ -12,6 +12,8 @@
     "ionic:serve": "ionic-app-scripts serve"
   },
   "dependencies": {
+    "@aerogear/auth": "../packages/auth",
```

Comment:
are these correct?

Reply:
What you mean?

Reply:
ah. you are running this app from within the sdk directory are you? I am running it independently - I assumed this was how we were going to do it

Reply:
I just noticed the script in the showcase app which tells me you are. I thought the idea was to loosely couple the two, I think when we release each package as an npm module it will matter less but I wouldn't _assume_ that the app will be run from within the sdk.

Reply:
This is fix before we release them. It in fact changes the flow for developers only. For now you can just checkout SDK and you will get example bty script. This will help internally to develop on top of the example.